CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+## JavaScript
+- Fixed a bug where if any injected dependencies to a JavaScript (e.g. `ux:Name`, `ux:Dependency`, `dep:`) have names that correspond to EcmaScript 5.1 reserved words, these dependencies will now be available with a dollar sign `$` prefix (e.g. `$undefined`) instead of causing a script compile error.
```

Comment:
If you are suggesting injecting `$foo` *in addition* to injecting `foo`, then it would be backwards compatible. However, I question the usefulness of this. JS programmers must be expected to know the language they are programming in.